persistence/src/main/java/com/softserve/borda/repositories/BoardColumnRepository.java

```diff
@@ -2,8 +2,20 @@
 
 import com.softserve.borda.entities.BoardColumn;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
 
 @Repository
 public interface BoardColumnRepository extends JpaRepository<BoardColumn, Long> {
+    List<BoardColumn> getAllBoardColumnsByBoardId(Long boardId);
+
+    @Transactional
+    @Modifying
+    @Query("update board_columns board_column set board_column.boardId = :boardId where board_column.id = :boardColumnId")
+    Integer setBoardIdForBoardColumn(@Param("boardColumnId") Long boardColumnId, @Param("boardId") Long boardId);
```

Comment:
you can do only CRUD operations
C - create
R - read/find/select
U - update
D - delete/remove

I think it is good to use `update` instead of `set` keyword here.
Same case applies for method above.

Reply:
Got it, thanks. Fixed both